package.json

```diff
@@ -9,7 +9,7 @@
   },
   "engines": {
     "node": ">= 14",
-    "yarn": "please-use-npm"
+    "yarn": "\n\n\n\u001b[1;31m[ERROR]\u001b[32m Please use npm instead!\u001b[0m\n\n\n"
```

Comment:
Actually, please could you test this also on white (light) console, just to align color so it's OK-ish on both.

Reply:
Here's white with 2 spaces, in regular green then in bold green.

![image](https://user-images.githubusercontent.com/18623773/100361026-90544e00-2ff1-11eb-8903-0b1a4dba053a.png)

Reply:
The same in black

![image](https://user-images.githubusercontent.com/18623773/100361345-f93bc600-2ff1-11eb-8d23-39b9bd30e4c1.png)

Reply:
Any is fine - yet again your call 🙈

Reply:
Went with 2 spaces and bold green.